homeassistant/components/shopping_list/__init__.py

```diff
@@ -53,10 +55,23 @@
 )
 
 
-@asyncio.coroutine
-def async_setup(hass, config):
+async def async_setup(hass, config):
     """Initialize the shopping list."""
 
+    if not hass.config_entries.async_entries(DOMAIN):
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN, context={"source": config_entries.SOURCE_IMPORT}
+            )
+        )
+
+    return True
+
+
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax. Please drop this decorator and prefix async methods with `async` like `async def async_setup_platform`. Replace any `yield from` with `await`.